sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/src/AzureMonitorTransmitter.cs

```diff
@@ -39,26 +39,30 @@ public async ValueTask<int> TrackAsync(IEnumerable<TelemetryItem> telemetryItems
                 return 0;
             }
 
-            Azure.Response<TrackResponse> response = null;
+            int itemsAccepted = 0;
 
             try
             {
                 if (async)
                 {
-                    response = await this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false);
+                    itemsAccepted = await this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).ConfigureAwait(false);
                 }
                 else
                 {
-                    response = this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).Result;
+                    itemsAccepted = this.applicationInsightsRestClient.InternalTrackAsync(telemetryItems, cancellationToken).Result;
                 }
             }
             catch (Exception ex)
             {
+                if (ex?.InnerException?.InnerException?.Source == "System.Net.Http")
```

Comment:
@pakrym what's the best practice for Azure SDKs? Relying on the knowledge of inner exception hierarchy seems fragile and could break in the future.

Reply:
Will merge the PR and follow up with @pakrym for best practice.